### ISSUE TITLE
[Bug]: Error during "composer install"

### ISSUE THREAD

**Issue by MGeurts** [tasteui/tasteui#567]
### What Happened

```
Loading composer repositories with package information
Updating dependencies
Nothing to modify in lock file
Installing dependencies from lock file (including require-dev)
Nothing to install, update or remove
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   Error

  Attribute "TallStackUi\Foundation\Attributes\SkipDebug" cannot target parameter (allowed targets: property)

  at vendor\laravel\framework\src\Illuminate\Container\Container.php:1369
    1365▕      */
    1366▕     protected function fireAfterResolvingAttributeCallbacks(array $attributes, $object)
    1367▕     {
    1368▕         foreach ($attributes as $attribute) {
  ➜ 1369▕             if (method_exists($instance = $attribute->newInstance(), 'after')) {
    1370▕                 $instance->after($instance, $object, $this);
    1371▕             }
    1372▕
    1373▕             $callbacks = $this->getCallbacksForType(

  1   vendor\laravel\framework\src\Illuminate\Container\Container.php:1369
      ReflectionAttribute::newInstance()

  2   vendor\laravel\framework\src\Illuminate\Container\Container.php:1024
      Illuminate\Container\Container::fireAfterResolvingAttributeCallbacks()
```

### Component

Alert

### How to Reproduce

Update TallstackUI to the latest version.

### Laravel Version

11.x

### Livewire Version

3.3

### TallStackUI Version

1.33.0

### PHP Version

8.3.9

### Operation System

Windows

**Comment by devajmeireles** [comment]
Hey! Thanks for reporting. This seems strange to me. Are you customizing something deeply?

**Comment by MGeurts** [comment]
All worked well for the last 3 months.
I just updated the folowing stuff when the error appeared:

```
  - Upgrading laravel/fortify (v1.21.5 => v1.22.0)
  - Upgrading laravel/framework (v11.16.0 => v11.17.0)
  - Upgrading laravel/jetstream (v5.1.3 => v5.1.4)
  - Upgrading laravel/pint (v1.16.2 => v1.17.0)
  - Upgrading laravel/sail (v1.30.2 => v1.31.0)
```
Rolling back the updates makes the error dissapear.

But I wonder if it has to do with TallStackUI due to `TallStackUi\Foundation\Attributes\SkipDebug`

**Comment by MGeurts** [comment]
The only thing that's out of the normal is my composer.json 👍 

```
  "post-autoload-dump": [
      "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
      "@php artisan package:discover --ansi",
      "@php artisan filament:upgrade",
      "@php artisan tallstackui:setup-icon --force --ansi"
  ],
```

**Comment by vgoncalves13** [comment]
I'm receiving this error too.

Appear after i composer update

**Comment by vgoncalves13** [comment]
I downgrade my laravel to 11.16 and my system back to work.

**Comment by devajmeireles** [comment]
Thanks for reporting, @MGeurts ! This issue is related to Laravel. I reported here: https://github.com/laravel/framework/issues/52247

**Comment by devajmeireles** [comment]
The fix in TallStackUI seems easy. I'll just be waiting for Laravel's position to see if they will rollback or not.

**Comment by devajmeireles** [comment]
Fixed now in v1.33.1: https://github.com/tallstackui/tallstackui/releases/tag/v1.33.1.

Run a `composer update` to get this new version.
